rewrite-java-core/src/main/resources/META-INF/rewrite/rewrite.yml

```diff
@@ -10,7 +10,30 @@ recipeList:
   #    find: 'requires\s+(transitive\s+)?com\.azure\.core(?!\.v2)'
   #    replace: 'requires $1com.azure.core.v2'
   #    filePattern: '**/module-info.java'
+
+  # ----------------------------------
+  #
+  # Java-Based LST Replacement Recipes
   #
+  # ----------------------------------
+
+  # Recipes that converts the parameters and type of the @UnexpectedResponseExceptionType annotation
```

Comment:
We don't need to convert annotations. Users don't need to use the annotations directly. Annotations are only needed by the Azure libraries. So, we don't need to update them using OpenRewrite recipes.

Reply:
Removed recipes that convert annotations and their associated unit tests in latest commit.